src/App.js

```diff
@@ -160,7 +160,7 @@ const routeConfig = [
   { path: "/AnimalsDash/AnimalsMeowFacts", component: AnimalsMeowFacts },
   { path: "/AnimalsDash/AnimalsRandomDog", component: AnimalsRandomDog },
   { path: "/AnimalsDash/AnimalsShibeOnline", component: AnimalsShibeOnline },
-  { path:"/AnimalsDash/AnimalCatFacts", component:Catfacts},
+  { path:"/AnimalsDash/AnimalCatFacts", component:AnimalCatFacts},

```

Comment:
Animals

Reply:
Sorry my Bad I have made the necessary changes